merge_requests.go

```diff
@@ -111,22 +80,29 @@ type MergeRequest struct {
 		RenamedFile bool   `json:"renamed_file"`
 		DeletedFile bool   `json:"deleted_file"`
 	} `json:"changes"`
-	TimeStats *TimeStats `json:"time_stats"`
-	Squash    bool       `json:"squash"`
-	Pipeline  struct {
-		ID     int    `json:"id"`
-		Ref    string `json:"ref"`
-		SHA    string `json:"sha"`
-		Status string `json:"status"`
-	} `json:"pipeline"`
-	DiffRefs struct {
+	TimeStats    *TimeStats    `json:"time_stats"`
+	Squash       bool          `json:"squash"`
+	Pipeline     *PipelineInfo `json:"pipeline"`
+	HeadPipeline *Pipeline     `json:"head_pipeline"`
```

Comment:
Are you sure all the fields of `*PipelineInfo` and `*Pipeline` match up to the `Pipeline` and `HeadPipeline` fields?

Reply:
Absolutely. The example above is used for reference, but you can see from the code (lib/api/entities.rb:790-796) that these entities are exposed using the formatters used in the pipelines API. This reuse of code on the backend mirrors this reuse here.

Reply:
Cool 👍